tuxemon/map_loader.py

```diff
@@ -66,23 +67,46 @@ def load_events(self, path: str) -> Iterator[EventObject]:
             path: Path to the file.
 
         """
+        yaml_data: dict[str, dict[str, dict[str, Any]]] = {}
+
         with open(path) as fp:
             yaml_data = yaml.load(fp.read(), Loader=yaml.SafeLoader)
 
         for name, event_data in yaml_data["events"].items():
+            _id = uuid.uuid4().int
             conds = []
             acts = []
-            x = event_data.get("x")
-            y = event_data.get("y")
-            w = event_data.get("width")
-            h = event_data.get("height")
+            x = (
+                1
+                if event_data.get("x") is None
+                else (event_data.get("x", int))
+            )
+            y = (
+                1
+                if event_data.get("y") is None
+                else (event_data.get("y", int))
+            )
+            w = (
+                0
```

Comment:
I didn't know an event could have a width or height of 0.
I assume you tested this?

Reply:
right, you're right, I missed it